File: SEM3_CODE/PHP/BASIC/20_function.php
<?php
function greet($name) {                     // Define a greeting function
    $message = 'Hello, ' . $name;           // Create the greeting message
    return $message;                        // Return the message
}

$userName = readline('Enter your name: ');  // Read the user's name
$result = greet($userName);                 // Call the function
echo $result;                               // Display the returned message
?>

File: SEM5_CODE/PYTHON/BASIC/20_function.py
def greet(name):                        # Define a greeting function
    message = "Hello, " + name          # Create the greeting message
    return message                      # Return the message


user_name = input("Enter your name: ")  # Read the user's name
result = greet(user_name)               # Call the function
print(result)                           # Display the returned message
